server/webSocketServer/index.js

```diff
@@ -1,38 +1,43 @@
 const WebSocket = require('ws');
-const userContainer = require('./userContainer');
+const statusTimer = require('./UserStatus/statusTimer');
```

Comment:
Rename to statusTimerFactory or something

Reply:
Nice, I like it